quadkey/sf/test/bbox_integration.js

```diff
@@ -0,0 +1,72 @@
+const assert = require('assert').strict;
+const snowflake = require('snowflake-sdk');
+
+const SF_DATABASEID = process.env.SF_DATABASEID;
+const SF_SCHEMA_QUADKEY = process.env.SF_SCHEMA_QUADKEY;
+
+function execAsync(connection, sqlText) {
+    return new Promise((resolve, reject) => {
+        connection.execute({
+            sqlText: sqlText,
+            complete: (err, stmt, rows) => {
+                if (err) {
+                    return reject(err);
+                } 
+                return resolve([stmt, rows]);
+            }
+        });
+    });
+}
+
+describe('BBOX integration tests', () => {
+    let connection;
+    before(async () => {
+        if (!SF_DATABASEID) {
+            throw "Missing SF_DATABASEID env variable";
+        }
+        if (!SF_SCHEMA_QUADKEY) {
+            throw "Missing SF_SCHEMA_QUADKEY env variable";
+        }
+        connection = snowflake.createConnection( {
```

Comment:
I strongly dislike having username and passwords in the environment. Are there other options to handle this, like config files and so on? Could something like https://docs.snowflake.com/en/user-guide/snowsql-start.html#configuring-default-connection-settings be used instead and remove the need to declare any login parameters?

Reply:
Not fan at all of using username/pass as credentials neither. It's possible to use a config file for the "snowsql" CLI client as you have sent me. I also found a possible way to set this config file automatically in the CI:
https://github.com/gvnirmalaa/ansible-role-snowsql

The main problem here is the snowflake NodeJs Driver which is more cumbersome to authenticate. Of the different options, user/pass looked like the simplest, more familiar to possible contributors and also reusable in the CLI.
https://docs.snowflake.com/en/user-guide/nodejs-driver-use.html#setting-the-connection-options

Anyway will ask the snowflake team, since in the config file we can set different connections with their credentials. And would be nicer to do something like: snowflake.createConnection({CONFIG_FILE_PATH, CONEXION_NAME}). Or even just send a file path and take a default conexion.

Reply:
As discussed today, we are going for the user/account/pass solution